.github/workflows/ci.yml

```diff
@@ -13,14 +13,18 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
         django-version: ["4.2", "5.0"]
-        wagtail-version: ["5.2", "6.0", "6.1"]
+        wagtail-version: ["5.2", "6.2"]
+        include:
+          - python-version: "3.12"
+            django-version: "5.1"
+            wagtail-version: "6.3"
```

Comment:
Question: Why is Wagtail 6.3 only being tested on very specific Python and Django versions?

Reply:
Oh yeah I could include 3.10 and 3.11 as well. I'll do an update.

Reply:
I've switched it over to use exclude, thanks